src/citrine/ara/variables.py

```diff
@@ -101,3 +110,232 @@ def __init__(self, *,
         self.name = name
         self.headers = headers
         self.template = template
+
+
+class AttributeByTemplateAfterProcessTemplate(Serializable['AttributeByTemplateAfterProcessTemplate'], Variable):
+    """Attribute of an object that is marked by an attribute template and derives from a process
+    marked by a locally unique object template.
```

Comment:
```suggestion
    marked by a locally unique process template.
```
It's also an object template, but might as well be as specific as possible.